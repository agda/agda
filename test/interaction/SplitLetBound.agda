open import Agda.Builtin.Bool

f : Bool â†’ Bool
f x = let y = x in {!!}
