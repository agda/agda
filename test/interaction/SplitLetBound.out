(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Bool " nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-info-action "*Error*" "SplitLetBound.agda:4,20-24 Cannot split on let-bound variable y when checking that the expression ? has type Bool" nil)
((last . 3) . (agda2-maybe-goto '("SplitLetBound.agda" . 67)))
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
